src/st_stuff.c

```diff
@@ -677,6 +679,30 @@ void ST_updateFaceWidget(void)
 }
 
 int sts_traditional_keys; // killough 2/28/98: traditional status bar keys
+int hud_blink_keys; // [crispy] blinking key or skull in the status bar
+
+void ST_BlinkKeys(player_t* player, int blue, int yellow, int red)
+{
+  int i;
+  int keys[3] = { blue, yellow, red };
+
+  player->keyblinktics = KEYBLINKTICS;
+
+  for (i = 0; i < 3; i++)
+  {
+    if (   ((keys[i] == KEYBLINK_EITHER) && !(player->cards[i] || player->cards[i+3]))
```

Comment:
Nitpick, but could you move the `||` at the end of each line, please?

Reply:
Like this?

```c
    if (((keys[i] == KEYBLINK_EITHER) && !(player->cards[i] || player->cards[i+3])) ||
        ((keys[i] == KEYBLINK_CARD)   && !(player->cards[i])) ||
        ((keys[i] == KEYBLINK_SKULL)  && !(player->cards[i+3])) ||
        ((keys[i] == KEYBLINK_BOTH)   && !(player->cards[i] && player->cards[i+3])))
```

I feel like it harms readability, but if you insist, so be it.

Reply:
No, I don't insist. It's just a different style after all.